src/Compilers/Test/Core/AssemblyLoadTestFixture.cs

```diff
@@ -385,6 +389,64 @@ public int GetFileAttributes(string path)
 }
 
 ");
+
+
+            var analyzerWithFakeCompilerDependencyDirectory = _directory.CreateDirectory("AnalyzerWithFakeCompilerDependency");
+            var fakeCompilerAssembly = GenerateDll("Microsoft.CodeAnalysis", analyzerWithFakeCompilerDependencyDirectory, @"
+using System;
+using System.Reflection;
+
+[assembly: AssemblyVersionAttribute(""42.42.42.42"")]
```

Comment:
Why use 42.42.42.42 here? This will sometimes match the version in the MS.CA running in this process and sometimes not match it. Feels like that could lead to flakiness down the road (running unit tests on official binaries for example). My instinct says a different number should be here. Am I missing something about what this helps with?